Dockerfile.test

```diff
@@ -0,0 +1,12 @@
+FROM python:3.8-slim-buster
+WORKDIR /app
+
+RUN pip install poetry \
+  && poetry config virtualenvs.create false
+
+COPY poetry.lock pyproject.toml /app/
+RUN poetry install
+
+COPY . /app
+# We don't do e2e tests yet
```

Comment:
What's the advised way of doing e2e tests with Docker? Is it e.g. using compose to spin up the app and also a geckodriver thing?